src/Commands/Integration.ts

```
@@ -96,7 +106,8 @@ export class Integration implements CommandModule {
     const startupManagerFactory: StartupManagerFactory = container.resolve(
       StartupManagerFactory
     );
-    const integrationClient: IntegrationClient = container.resolve(IntegrationClient);
+    const pingTracers: IntegrationPingTracer[] = container.resolveAll(IntegrationClient);
+    const pingTracer = pingTracers.find((p) => p.type === args.type);
```

When `pingTracer` is undefined, you should throw an error with an explanation